sdk/storage/azdatalake/directory/client_test.go

```diff
@@ -767,6 +769,72 @@ func (s *RecordedTestSuite) TestDeleteDirWithNilAccessConditions() {
 	_require.NotNil(resp)
 }
 
+func (s *UnrecordedTestSuite) TestDeleteDirWithPaginatedDelete() {
```

Comment:
Test will be recorded, but it will be skipped by default, because it requires a customer AAD app that don't need to be configured in live test pipelines. 